### DISCUSSION TITLE
From the sdk Can i specify the locale and get multi language support?

### DISCUSSION THREAD

**Discussion by Nanagoud007** [flutter-stripe/flutter_stripe#1446] [Q&A]
**Is your feature request related to a problem? (please describe)**
A clear and concise description of what the problem is. Ex. I would like to see [...]

**Describe the solution you'd like**

**Describe alternatives you've considered**
A clear description of  alternative solutions or features you've considered.

**Additional context**

**Comment by remonh87** [comment]
When you use the paymentsheet localization is build in automatically. If you use cardfield and cardform localization is limited
